crates/services/bls_aggregation/src/bls_agg.rs

```diff
@@ -134,21 +129,47 @@ impl TaskSignature {
     }
 }
 
+// Messages to be sent to BLS Aggregator Service
+pub enum AggregationMessage {
+    InitializeTask(TaskMetadata),
+    ProcessSignature(TaskSignature),
+}
+
+// Handler to interact with the BLS Aggregator Service
+#[derive(Debug, Clone)]
+pub struct ServiceHandler {
```

Comment:
"handle" is a common name for this kind of concept.

```suggestion
pub struct ServiceHandle {
```